src/views/TopicView.tsx

```diff
@@ -4,7 +4,11 @@ import React from "react";
 import { StyleSheet, Text, View } from "react-native";
 import { ScrollView } from "react-native-gesture-handler";
 
-export const TopicView: React.FC = () => {
+export type TopicViewProps = {
+  tid: string;
+};
+
+export const TopicView = ({ tid }: TopicViewProps) => {
```

Comment:
Why are `TopicView` and `TopicScreen` just share the same props?

Reply:
I believe that TopicView and TopicScreen are two different components. Although their variables are currently named "id," in the future they may have separate variables added for each component. Additionally, for the sake of adhering to naming conventions and ensuring clarity in parameter interfaces, it would be better to define them separately.